libr/bin/demangle.c

```diff
@@ -91,6 +91,13 @@ R_API char *r_bin_demangle(RBinFile *bf, const char *def, const char *str, ut64
 				break;
 			}
 		}
+		size_t len = strlen (bin->file);
+		if (!r_str_ncasecmp (str, bin->file, len)) {
```

Comment:
why is the filename affecting this the demangling output? can you show a screenshot about the issue?

Reply:
@radare The filename is included in the symbol name:
https://github.com/radare/radare2/blob/7d30ff52fc1be6f9698b166107b8981eab6ec7ba/libr/bin/format/pe/pe.c#L3025

 but isn't on the libs list, so it doesn't get skipped
ie: `GameLogic.dll_?HideCountdown@IPlayer@@QAEXXZ`

Would it be better to change that line and remove the library name from the symbol altogether?
`?HideCountdown@IPlayer@@QAEXXZ`

Reply:
![image](https://user-images.githubusercontent.com/23562137/62091785-10757280-b249-11e9-8fe7-28ce666fc587.png)
vs

![image](https://user-images.githubusercontent.com/23562137/62091880-72ce7300-b249-11e9-9f5f-88c397e58a23.png)

Reply:
I remember @Maijin asked to include library name to the symbol.

Reply:
? Nop, I said, we could always keep it under an eval because it can be useful time to time when there is confusion onto where one comes from.

Reply:
@GustavoLCR can do do the change as suggested by @Maijin ?